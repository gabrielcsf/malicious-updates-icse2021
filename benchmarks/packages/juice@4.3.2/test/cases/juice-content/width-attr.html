<html>
<head>
    <style>
        p, td.px, img.px {
            width: 200px;
        }
        td.percentage, img.percentage {
            width: 50%;
        }
    </style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td class="px">
                wide
            </td>
            <td class="percentage">
                wide
            </td>
        </tr>
    </table>
    <img src="" alt="wide" class="px">
    <img src="" alt="wide" class="percentage">
</body>
</html>
