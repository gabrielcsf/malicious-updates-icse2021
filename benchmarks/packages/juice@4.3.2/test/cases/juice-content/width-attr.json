{
    "url": "./",
    "removeStyleTags": true,
    "applyWidthAttributes": true
}