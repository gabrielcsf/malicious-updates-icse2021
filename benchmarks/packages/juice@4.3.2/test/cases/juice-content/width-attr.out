<html>
<head>
    
</head>
<body>
    <p style="width: 200px;">none</p>
    <table>
        <tr>
            <td class="px" style="width: 200px;" width="200">
                wide
            </td>
            <td class="percentage" style="width: 50%;" width="50%">
                wide
            </td>
        </tr>
    </table>
    <img src="" alt="wide" class="px" style="width: 200px;" width="200">
    <img src="" alt="wide" class="percentage" style="width: 50%;">
</body>
</html>
